### ISSUE TITLE
Feat: Develop homepage for the website

### ISSUE THREAD

**Issue by Rahulm2310** [anitab-org/open-source-programs-web#97]
### Is your feature request related to a problem? Please describe.

As a user, I need a home page for the website to know more and get a better clarity about the project.

### Describe the solution you'd like

Develop the home page for the website.

### Additional context

Mock Up for reference: 
![87044100-b54b4e80-c213-11ea-9e8e-6ef64f242585](https://user-images.githubusercontent.com/54268438/109050307-3f8ee700-76ff-11eb-9ac6-a416fdea95f9.png)
Reference: #31

**Comment by Rahulm2310** [comment]
@isabelcosta I would like to work on this. 🙂

**Comment by isabelcosta** [comment]
@Aaishpra @codesankalp this is not a priority, but can still be done 🤷🏾‍♀️ what do you think? If yey, then please assign this :)

**Comment by Aaishpra** [comment]
@isabelcosta from the discussion in #31 I came to know that osp isn't following any design/margin guidelines (means the above design was made without keeping in mind the AnitaB design and colour guidelines) and at some point of time in future(when the MVP is completed) I think we might want to have some specific design guidelines for the project (like BIT) which would mean redesigning of this page(if this page is created at the present) along with  designing of other pages, which will cost us double the amount of work and time (similar thing is happening with the current homepage in the BIT). Along with this the design is quite old and I guess we might need the @anitab-org/designers approval with it. 
And the content for the homepage is not yet finalized (we can't add lorem ispum to the homepage) we need @mayburgos review in #30 means we cannot go forward with this issue until the content is finalized. 
I think this issue should be put to hold until the MVP is completed and should be revisited again when we have proper content and design guidelines for the project.
Although this is just my opinion and if you want we can have a discussion on zulip with the design team members about this and then go forward with it.

Cc @isabelcosta

**Comment by codesankalp** [comment]
Yes, @isabelcosta I totally agree with @Aaishpra , we need to discuss #31 before this issue due to the reasons mentioned by @Aaishpra and the Logo is also not present in the design.
Although the homepage is also an important part of the website, we need approved designs before moving on to developing the website homepage as it is the main focus of the users (so better UX & UI is needed).
So my final opinion is to tackle this issue after #31.

**Comment by isabelcosta** [comment]
Ok, let's ask for help of design team for this then :) Thank you for your inputs! I put the label as on hold

**Comment by NegiAkash890** [comment]
I would like to contribute when the design gets finalized/approved .
 Thank You

**Comment by Diksha4111** [comment]
Can I get the opportunity to work on this issue?

**Comment by codesankalp** [comment]
> Can I get the opportunity to work on this issue?

@Diksha4111 This issue is follow up of #31. If you can share any design on that issue, then surely after that you can start working on it.

**Comment by Diksha4111** [comment]
@codesankalp Thanks for showing interest on my request. I will surely share a design with you 😊

**Comment by Mrunal2714** [comment]
I am interested on working on this issue

**Comment by aryan-bais** [comment]
i would like to work on this
